crates/core_arch/src/mod.rs

```diff
@@ -162,12 +173,12 @@ pub mod arch {
     /// > `-Ctarget-feature=+simd128,+unimplemented-simd128`. This second
     /// > feature enables more recent instructions implemented in LLVM which
     /// > haven't always had enough time to make their way to runtimes.
-    #[cfg(any(target_arch = "wasm32", doc))]
-    #[doc(cfg(target_arch = "wasm32"))]
-    #[stable(feature = "simd_wasm32", since = "1.33.0")]
-    pub mod wasm32 {
-        #[stable(feature = "simd_wasm32", since = "1.33.0")]
-        pub use crate::core_arch::wasm32::*;
+    #[cfg(any(target_arch = "wasm32", target_arch = "wasm64", doc))]
+    #[doc(cfg(any(target_arch = "wasm32", target_arch = "wasm64")))]
+    #[stable(feature = "", since = "0.0.0")]
+    pub mod wasm {
+        #[stable(feature = "", since = "0.0.0")]
```

Comment:
This should probably be private. Even `x86` and `x86_64` are only available on the respective targets despite the fact that `x86_64` re-exports everything in `x86`.

Reply:
What i was trying to do here was avoid people having to write
```rust
#[cfg(target_os = "wasm32")]
use core::arch::wasm32 as wasm;
#[cfg(target_os = "wasm64")]
use core::arch::wasm64 as wasm;
```

Reply:
This doesn't match what we do for `x86`/`x86_64` right now, however, where you only get one or the other. I think that we'll want to stick to a `wasm32` and a `wasm64` module for now to mirror that. The `core::arch` module isn't optimized for ergonomic usage, but rather optimized for developers (us) to add intrinsics in a known location.